service/src/main/java/com/ge/predix/acs/attribute/connector/management/AttributeConnectorServiceImpl.java

```diff
@@ -176,9 +180,23 @@ private void validateAdapterEntityOrFail(final AttributeAdapterConnection adapte
         if (adapter.getAdapterEndpoint() == null || adapter.getAdapterEndpoint().isEmpty()) {
             throw new AttributeConnectorException("Attribute adapter configuration requires a nonempty endpoint URL");
         }
+        try {
+            if (!new URI(adapter.getAdapterEndpoint()).getScheme().equalsIgnoreCase(HTTPS)) {
```

Comment:
I think we should replace this line (new URI(adapter.getAdapterEndpoint()).getScheme().equalsIgnoreCase(HTTPS)) with function in Utils with can be used in multiple places. Also we need to handle the case if scheme is not specified(for example if someone specifies www.abcd.com)